sentry/src/main/java/io/sentry/util/StringUtils.java

```diff
@@ -51,4 +54,22 @@ private StringUtils() {}
       return str;
     }
   }
+
+  /**
+   * Converts the given number of bytes to a human-readable string.
+   *
+   * @param bytes the number of bytes
+   * @return a string representing the human-readable byte count (e.g. 1kB, 20 MB, etc.)
+   */
+  public static @NotNull String byteCountToString(long bytes) {
```

Comment:
```suggestion
  public static @NotNull String byteCountToString(final long bytes) {
```

Reply:
Can't do that since we modify the bytes in line 70